docs/r3/beyondcandc.md

```diff
@@ -1,28 +1,29 @@
 
-The establishment of suitable command-and-control methods for systems to query, instruct and observe one another is necessary but not sufficient for interoperation when the requirements include reliable synchronisation of state changes across networks without a trusted central co-ordinator. Building on these primitives, we also require:
+The establishment of suitable command-and-control methods for systems to query, instruct and observe one another is _necessary but not sufficient_ for interoperation when requirements include reliable synchronisation of state changes across networks without a trusted central co-ordinator. Building on these primitives, we also require:
 
 * **Proof of action**: the ability to construct and transmit across network boundaries proof that an action meeting specific criteria has been taken on a network.
-* **Offline verification**: the ability to check a proof originating on a network, or from a part of a network, to which the checker does not have ongoing access.
+* **Offline verification**: the ability to check a proof originating on a network, or part of a network, to which the checker does not have ongoing access.
 
 These requirements are more subtle. Suppose we take as proof of action a digital record of a transaction signed by known validators whose signatures attest that it was finalised on its network's ledger. To know whether the record describes the specific action we need to verify, we must be able to interpret its data structure. Because validators typically sign over a network’s internal representations of its ledger transactions, we risk having to build a detailed model of one network’s ledger representation inside the smart contract logic of another.
 
 To help us think more concretely about these questions, we now consider an example use case: the cross-network swap.
 
 ## Example: cross-network swap
 
-Our motivating example here is a cross-network swap, in which party P on network A transfers an asset to party Q on network A, in exchange for party Q' transferring an asset to party P' on network B.
+Our motivating example here is a cross-network swap, in which Alice on a network managing tokens with a dollar value (Alice@USD) transfers funds to Bob@USD, in exchange for Bob's counterpart on a network managing tokens with a sterling value (Bob@GBP) transferring an asset to Alice's counterpart on that network (Alice@GBP).
 
-We make an assumption that P and P' have a common interest (for example, both represent token wallets held by the same commercial entity), as do Q and Q'.
+We make an assumption that Alice@USD and Alice@GBP have a common interest (for example, both represent token wallets held by the same commercial entity), as do Bob@USD and Bob@GBP. Although the common naming may suggest they are the "same" identity, there is no global identity model containing an entity to which they both refer.
```

Comment:
I woner if this isn't more complicated than casting the example in the reverse i.e. simply stating that Alice@X and Alie@Y are the same human (comapny)